R/layer.R

```diff
@@ -304,7 +302,7 @@ Layer <- ggproto("Layer", NULL,
 
     # Evaluate aesthetics
     evaled <- eval_aesthetics(aesthetics, data)
-    plot$scales$add_defaults(evaled, plot$plot_env)
+    plot@scales$add_defaults(evaled, plot@plot_env)
```

Comment:
no fault of yours, but this mix of `@` and `$` indexing is really aesthetically unpleasant

Reply:
Agreed. It wouldn't be that much of a problem to implement `@.ggproto` I think, so we can eliminate one source of annoyance.

Reply:
I hear you